setup.cfg

```diff
@@ -1,3 +1,52 @@
+[metadata]
+name = pytd
+version = 1.4.0
```

Comment:
Unfortunately `version = attr: pytd.__version__` doesn't work well while there was improvement for it
https://stackoverflow.com/questions/58202909/modulenotfounderror-when-using-setup-cfg-and-version-accessed-with-attr
https://github.com/pypa/setuptools/pull/1753